ui/src/main/webapp/src/app/app.routing.ts

```diff
@@ -98,7 +98,7 @@ export const appRoutes: Routes = [
                             wizard: true
                         },
                         children: [
-                            { path: 'create-project', component: MigrationProjectFormComponent, data: {displayName: 'Create new Project', wizard: true} },
+                            { path: 'create-project', component: MigrationProjectFormComponent, data: {displayName: 'Create new Project', wizard: true, creating: true} },
```

Comment:
Right, it really duplicates. I overlooked.
Anyway, isn't this better, as it spans the flag over the whole wizard?